tests/test_pull_model.py

```
@@ -0,0 +1,159 @@
+        "pull-model.ps1 נשמר בלי BOM — Windows PowerShell 5.1 יקרא אותו כ-ANSI"
+    )
+
+
+def test_the_script_stays_under_the_line_limit():
+    lines = source().count("\n") + 1
+    assert lines <= MAX_LINES, f"pull-model.ps1: {lines} שורות"
+
+
+def test_errors_are_not_swallowed_into_a_pass():
+    """עיקרון 5 כטקסט. ‏Get-Command הוא היוצא מן הכלל היחיד, ושם
+    ההיעדר עצמו מדווח ככישלון — לא מומר להצלחה."""
```

Ruff (0.16.3)

[error] Contains control characters that can permit obfuscated code

(PLE2502)